pkg/cmd/prebuild/update.go

```diff
@@ -19,20 +18,31 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
```

Comment:
We should move this flag block just above the `init` function in order to be consistent with the rest of the codebase

Reply:
moved just above the init function